torchrecipes/audio/source_separation/conf/train_app.yaml

```diff
@@ -0,0 +1,16 @@
+_target_: torchrecipes.audio.source_separation.train_app.SourceSeparationTrainApp
+
+defaults:
+  - _self_
+  # module
+  # - schema/module: conv_tasnet_module_conf
```

Comment:
Is this commented out on purpose?

Reply:
Yes. @tangbinh is still working on new config system proposal. I will keep the current design by removing the whole line to proceed.